wa/instruments/perf.py

```diff
@@ -134,5 +146,38 @@ def update_output(self, context):
                                 metric = '{}_{}'.format(label, match.group(3))
                                 context.add_metric(metric, count, classifiers=classifiers)
 
+    def _update_output_record(self, context):
+        outdir = os.path.join(context.output_directory, 'perf')
+        self.collector.get_trace(outdir)
```

Comment:
Given that the above two lines happen in both `_stat` and `record` cases, they should be moved out into `update_output`.

Reply:
Thanks, changed in latest version.